src/styles/palette.js

```diff
@@ -69,6 +73,8 @@ module.exports = {
       shade60: 'rgba(255, 255, 255, 0.6)',
       shade40: 'rgba(255, 255, 255, 0.4)',
       shade20: 'rgba(255, 255, 255, 0.2)',
+      shade10: 'rgba(255, 255, 255, 0.1)',
+      shade04: 'rgba(255, 255, 255, 0.04)',
```

Comment:
rgba(255, 255, 255, 0.04) is really close from the default background, we should not add it as it is really specific and the theme will start to go out of hands again. let's use background.default !

if you have exceptional color needs that won't be used anywhere else, consider using the color library to darken / lighten it

Reply:
![image](https://user-images.githubusercontent.com/4631227/68410933-de4da800-0189-11ea-81e2-e70e9b439380.png)

Reply:
🤷‍♂

Reply:
Marouane is trying to cheat, but we need some consistency :) background.default is super close to .04 really

Reply:
I don't have an issue changing from `.04` to` background.default` I try to follow what the designer gives me and in this case it was an `.04` :trollface: 

Would it be possible to choose some names that are more consistent? I find it confusing that they are called "_text_" or "_divider_" and used for backgrounds if you know what I mean.

Reply:
@dekkaki

Reply:
Yes this is right, "text" and "divider" definitly came out of hand. I suppose "text" should be renamed and divider replaced by shade10. We'll improve the theme a bit soon :)